### ISSUE TITLE
Should /lib be checked into git?

### ISSUE THREAD

**Issue by tgriesser** [transduce/transduce#42]
The age old question of whether compiled files should be regen on release. If you're looking to support the absolute paths to files rather than name-spacing on a single build then I'd guess you may need to - unless you only check in at release time?

Makes PR's a bit of a pain to manage though

**Comment by kevinbeaty** [comment]
Yeah. I can't decide on this one.  I think we do want to generate the browser files in `build` on release.  Not sure about `lib` as those are there mostly for `npm` (and will be published there appropriately).

I could go either way.

**Comment by tgriesser** [comment]
The only real benefit of keeping things checked in is that you can reference any particular SHA commit in git in your package.json and it'll "just work", but this isn't a real issue if releases are cut frequently.

**Comment by kevinbeaty** [comment]
I'm leaning towards keeping it out for now, unless you have strong feelings about including them.. I'd like to avoid the downside you mentioned of making merge conflicts more difficult than necessary as they would need to be resolved in both places.  Hopefully it won't be too common that someone would want to pin to a specific commit.

**Comment by tgriesser** [comment]
Cool - I think once you sign off on the other LazyIterator patch I'd be :+1: on a new release.
